src/crimson/osd/pg.h

```diff
@@ -797,6 +797,9 @@ struct PG::do_osd_ops_params_t {
   uint64_t get_features() const {
     return features;
   }
+  bool has_flag(uint32_t flag) const {
```

Comment:
Do we still need to pull this from the actual message?

Reply:
do_osd_ops( ... do_osd_ops_params_t ..) seems only is called in internal client request, no error handle, no return message (void), so I don't think it need check pool quota.

Reply:
Add a comment:

```
  // Only used by InternalClientRequest, no op flags
```

Reply:
done.